Initialization/SetupBaseStructure.py

```
@@ -198,27 +198,12 @@ def CompleteSecurityInitializer(self, security: Security) -> None:
         elif security.Type in [SecurityType.Option, SecurityType.IndexOption]:
             # This is for options.
             security.SetFillModel(BetaFillModel(self.context))
-            # security.SetFillModel(MidPriceFillModel(self))
             security.SetFeeModel(TastyWorksFeeModel())
             security.PriceModel = OptionPriceModels.CrankNicolsonFD()
-            # security.set_option_assignment_model(NullOptionAssignmentModel())
+            # Disable option assignment for index options
+            if security.Type == SecurityType.IndexOption:
```

Ruff (0.8.0)

`SecurityType` may be undefined, or defined from star imports

(F405)